HW_1/task.py

```diff
@@ -0,0 +1,25 @@
+from flask import Flask
+from flask import render_template
+
+app = Flask(__name__)
+
+@app.route('/')
+@app.route('/index.html/')
+def index():
+    return render_template('index.html')
+
+@app.route('/hats.html/')
+def hats():
+    return render_template('hats.html')
+
+@app.route('/clothes.html/')
+def clothes():
+    return render_template('clothes.html')
+
+@app.route('/shoes.html/')
```

Comment:
Теперь можно сделать так
```suggestion
@app.route('/')
def index():
    return render_template('index.html')

@app.route('/hats')
def hats():
    return render_template('hats.html')

@app.route('/clothes')
def clothes():
    return render_template('clothes.html')

@app.route('/shoes')
```